app/AcquireIntersonArrayRF.cxx

```diff
@@ -119,20 +119,20 @@ int main( int argc, char * argv[] )
   hwControls.DisableHardButton();
 
   ContainerType container;
+  container.SetHWControls(&hwControls);
 
   const int height = hwControls.GetLinesPerArray();
```

Comment:
Agreed that staying close to the driver notation for consistency in the wrapper is a good approach.